.gitignore

```diff
@@ -12,6 +12,7 @@ test/coverage
 *.ntvs*
 *.njsproj
 *.sln
+*.swp
```

Comment:
I'd suggest to add this to your global gitignore: `~/.gitignore`

Reply:
Got it. I'll remove it.